packages/expo-random/src/__tests__/Random-test.ts

```diff
@@ -0,0 +1,15 @@
+import { mockProperty, unmockProperty } from 'jest-expo';
+
+import ExpoRandom from '../ExpoRandom';
+import * as Random from '../Random';
+
+it(`invokes native method correctly`, async () => {
+  mockProperty(ExpoRandom, 'getRandomIntegerAsync', jest.fn(async () => ''));
```

Comment:
Re-generate the jest-expo mocks. Also then the native method will already be a mocked and you can do:

`ExpoRandom.getRandomIntegerAsync.mockImplementation(async () => '');`